pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -421,6 +422,33 @@ func (r *reconcilerBase) deployment(ctx context.Context, dRef client.ObjectKey)
 	return deployObj, nil
 }
 
+func mountConfigMapValuesFiles(c *corev1.Container, valuesFrom []v1beta1.ValuesFrom) {
+	if len(valuesFrom) == 0 {
+		return
+	}
+
+	var valuesFiles []string
+	for _, vf := range valuesFrom {
+		envVarName := reconcilermanager.HelmConfigMapRef + "_" + vf.Name
+		valuesFiles = append(valuesFiles, envVarName)
+		c.Env = append(c.Env, corev1.EnvVar{
+			Name: envVarName,
+			ValueFrom: &corev1.EnvVarSource{
+				ConfigMapKeyRef: &corev1.ConfigMapKeySelector{
```

Comment:
Here I'm mounting the valuesFile as an environment variable into helm-sync, but I think it will actually be better to just pass to helm-sync the names/keys of the ConfigMaps and have helm-sync read the ConfigMaps to get the values itself. 

I realized that environment variables have a length limit of about 32,000 characters, and valuesFiles can be quite large.

Reply:
Can you mount them as files in the container?

Reply:
I can! That will probably be better. Thanks

Reply:
Done.